includes/admin.form.inc

```diff
@@ -115,13 +96,7 @@ function islandora_usage_stats_admin_settings() {
   $form['clear']['usage']['ds']['clear_ds_by_pid_submit'] = array(
     '#type' => 'submit',
     '#name' => 'clearDSByPIDSubmit',
-    '#default_value' => t('Clear datastream download count'),
-  );
-
-  $form['clear']['usage']['ds']['clear_ds_all'] = array(
-    '#type' => 'submit',
-    '#name' => 'clearDSAllSubmit',
-    '#default_value' => t('Clear all datastream download counts'),
+    '#default_value' => t('Clear datastream download log'),
   );
 
   return system_settings_form($form);
```

Comment:
Might be time to refactor this out a bit? This is some shenanigans with how it's using validate to set a bunch of stuffs.